external/emr-logservice/src/main/scala/org/apache/spark/streaming/aliyun/logservice/SimpleLogHubProcessor.scala

```diff
@@ -78,6 +78,12 @@ class SimpleLogHubProcessor(receiver: LoghubReceiver)
       log.getContentsList.foreach(content => {
         obj.put(content.getKey, content.getValue)
       })
+
+      val flg = group.GetFastLogGroup()
+      for (i <- 0 to flg.getLogsCount) {
+        obj.put("__tag__:".concat(flg.getLogTags(i).getKey), flg.getLogTags(i).getValue)

```

Comment:
some minor issues:
1. "to" => "until"
2. "flg.getLogTagsCount" but not "flg.getLogsCount"